Migration script/openIMIS migration latest.sql

```diff
@@ -5842,6 +5842,20 @@ BEGIN
 END
 GO
 
+-- OTC-8: Authorisation doesn't work properly
+-- Adds user profile rights to IMIS Administrator
+
+DECLARE @SystemRole INT
+SELECT @SystemRole = role.RoleID from tblRole role where IsSystem=64; --IMIS Administrator
+
+INSERT INTO [dbo].[tblRoleRight] ([RoleID], [RightID], [ValidityFrom], [ValidityTo], [AuditUserId], [LegacyID])
+	SELECT @SystemRole, RightIDToAdd, CURRENT_TIMESTAMP, NULL, NULL, NULL
+	FROM ( values (122000), (122001), (122002), (122003), (122004), (122005)) as RightsToAdd (RightIDToAdd)) -- User Profile Rights
```

Comment:
Are all these rights already in the online script?

Reply:
Yes, these rights are in the online script. Lines 5052-5057
`INSERT [dbo].[tblRoleRight] ([RoleRightID], [RoleID], [RightID], [ValidityFrom], [ValidityTo], [AuditUserId], [LegacyID]) VALUES (89, 7, 122000, CURRENT_TIMESTAMP, NULL, NULL, NULL)
INSERT [dbo].[tblRoleRight] ([RoleRightID], [RoleID], [RightID], [ValidityFrom], [ValidityTo], [AuditUserId], [LegacyID]) VALUES (90, 7, 122001, CURRENT_TIMESTAMP, NULL, NULL, NULL)
INSERT [dbo].[tblRoleRight] ([RoleRightID], [RoleID], [RightID], [ValidityFrom], [ValidityTo], [AuditUserId], [LegacyID]) VALUES (91, 7, 122002, CURRENT_TIMESTAMP, NULL, NULL, NULL)
INSERT [dbo].[tblRoleRight] ([RoleRightID], [RoleID], [RightID], [ValidityFrom], [ValidityTo], [AuditUserId], [LegacyID]) VALUES (92, 7, 122003, CURRENT_TIMESTAMP, NULL, NULL, NULL)
INSERT [dbo].[tblRoleRight] ([RoleRightID], [RoleID], [RightID], [ValidityFrom], [ValidityTo], [AuditUserId], [LegacyID]) VALUES (93, 7, 122004, CURRENT_TIMESTAMP, NULL, NULL, NULL)
INSERT [dbo].[tblRoleRight] ([RoleRightID], [RoleID], [RightID], [ValidityFrom], [ValidityTo], [AuditUserId], [LegacyID]) VALUES (94, 7, 122005, CURRENT_TIMESTAMP, NULL, NULL, NULL)
`
`7` is the `IMIS Administrator`